.controlplane/templates/app.yml

```
@@ -28,3 +28,17 @@ spec:
 # Identity is needed to access secrets
 kind: identity
 name: {{APP_IDENTITY}}
+  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)